code/WorkInProgress/AzrunStuff.dm

```diff
@@ -0,0 +1,49 @@
+/datum/digital_filter
```

Comment:
documentation pls i bad at math

Reply:
Added brief description of use cases and relevant links to articles describing what is being implemented.

Let me know if you want to see more detail.